src/back-end/lib/resources/opportunity/code-with-us.ts

```diff
@@ -912,15 +911,6 @@ const update: crud.Update<
             }
             break;
           }
-          case "startEvaluation":
-            dbResult = await db.updateCWUOpportunityStatus(
-              connection,
-              request.params.id,
-              CWUOpportunityStatus.Evaluation,
-              body.value,
-              session
-            );
-            break;
```

Comment:
Is this no longer needed?

Reply:
It didn't seem like it was used anywhere in the code, I'm working on proposals in a separate PR so I might wait to see if those tests pass first